Bowling.java

```diff
@@ -117,6 +113,18 @@ public Integer getScore() {
 		return totalPoints;
 	}
 
+	public Boolean isGameOver() {
+		currentFrame = getCurrentFrame();
+		secondToLastFrame = getFrameBefore(currentFrame);
+		if (secondToLastFrame.isPresent()) {
+			thirdToLastFrame = getFrameBefore(secondToLastFrame.get());
+		}
+		
+		return allFrames.size() == 11 && 
+			   !secondToLastFrame.get().isStrike() && 
+			   !secondToLastFrame.get().isSpare();
```

Comment:
Not that I can recall - I let this slip out of my memory as well...let me see if I can re-load this morning.